docs/reference/contributing/target/entropy.md

```diff
@@ -88,9 +88,9 @@ To indicate that the target has an entropy source, you have to add `TRNG` to the
 
 ### How to implement the non-volatile seed entropy source
 
-If a hardware platform does not have a hardware entropy source to leverage into the entropy pool, alternatives have to be considered. As stated above, a strong entropy source is crucial for security of cryptographic and TLS operations. For platforms that support non-volatile memory, an option is to use the NV seed entropy source that Mbed TLS provides.
+If a hardware platform does not have a hardware entropy source to leverage into the entropy pool, alternatives have to be considered. As stated above, a strong entropy source is crucial for security of cryptographic and TLS operations. For platforms that support non-volatile memory, an option is to use the NV seed entropy source that Mbed provides.
 
-This makes Mbed TLS use a fixed amount of entropy as a seed and update this seed each time entropy is gathered with an Mbed TLS entropy collector for the first time. In a simple case it means that the seed is updated after reset at the start of the first TLS connection.
+This makes Mbed use a fixed amount of entropy as a seed and update this seed each time entropy is gathered with an Mbed entropy collector for the first time. In a simple case it means that the seed is updated after reset at the start of the first TLS connection.
```

Comment:
Maybe "This uses a fixed amount of entropy as a seed and updates this seed each time an entropy collector gathers entropy for the first time."?